dashboard/config/foorm/forms/surveys/pd/ayw_workshop_post_survey.0.json

```diff
@@ -25,18 +25,24 @@
       "name":"workshop_satisfaction_panel"
     },
     {
-      "type":"library_item",
-      "library_name":"surveys/pd/ayw_virtual_panel",
-      "library_version": 0,
-      "name":"ayw_virtual_panel"
-    }
-
+     "type": "panel",
+     "name": "ayw_post_virtual_panel",
+     "visibleIf":"{is_virtual}='true'",
+     "elements": [
+      {
+        "type":"library_item",
+        "library_name":"surveys/pd/ayw_virtual_panel",
+        "library_version": 0,
+        "name":"ayw_virtual_panel"
+      }
+    ]
+   }
```

Comment:
could we make this a page instead to keep the logic in this file? Since panels cannot currently contain library items

Reply:
Thanks, yeah.  I'm now remembering we discussed this but I forgot.  
Okay, made this a page for now in order to merge.  It makes for a lot of short pages in the survey, which I'm not a huge fan of.  But better to keep the logic here than punt it into the library item.  I don't think this goes live-live for a few weeks still.  So time to change if nec.